RecoTracker/FinalTrackSelectors/plugins/LwtnnESProducer.cc

```diff
@@ -0,0 +1,52 @@
+#include "FWCore/Framework/interface/ModuleFactory.h"
+#include "FWCore/Framework/interface/ESProducer.h"
+#include "FWCore/ParameterSet/interface/FileInPath.h"
+
+#include "RecoTracker/Record/interface/CkfComponentsRecord.h"
+
+//from lwtnn
+#include "lwtnn/LightweightNeuralNetwork.hh"
+#include "lwtnn/parse_json.hh"
+#include <fstream>
+
+class LwtnnESProducer: public edm::ESProducer {
+public:
+  LwtnnESProducer(const edm::ParameterSet& iConfig);
+  ~LwtnnESProducer() override = default;
+
+  static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
+
+  // TODO: Use of CkfComponentsRecord is as inadequate as the
+  // placement of this ESProducer in RecoTracker/FinalTrackSelectors
+  // (but it works, I tried to create a new record but for some reason
+  // did not get it to work). Especially if this producer gets used
+  // wider we should figure out a better record and package.
+  std::unique_ptr<lwt::LightweightNeuralNetwork> produce(const CkfComponentsRecord& iRecord);
```

Comment:
CkfComponentsRecord depends on many other records.
IIUC, this means that for any IOV changes that happen there all ES products in CkfComponentsRecord will be remade.
Is initialization of LightweightNeuralNetwork cheap enough for that?

Reply:
I would rather work on using some other record (CkfComponentsRecord was something I managed to get things working that we were able to submit the PR last week).

Should this use a new record or some existing one? If new record, where should it be placed? (as long as the ESProducer stays in RecoTracker, I guess RecoTracker/Record could be good-enough package)

We should probably move certain other algorithmic ESProducts out from CkfComponentsRecord as well (I recall (ab)using it before...).

Reply:
> Should this use a new record or some existing one?

From the available ones, perhaps TrackingComponentsRecord ? 

For a new one, something like TrackClassifiersRecord could be a good name